java/ingest/ingest-runner/src/test/java/sleeper/ingest/impl/IngestCoordinatorUsingDirectWriteBackedByArrowRecordWriterAcceptingRecordListIT.java

```diff
@@ -129,6 +232,41 @@ private List<RecordList> buildScrambledRecordLists(RecordGenerator.RecordListAnd
         return List.of(recordLists);
     }
 
+    private IngestCoordinatorTestParameters.Builder createTestParameterBuilder() {
+        return IngestCoordinatorTestParameters
+                .builder()
+                .temporaryFolder(temporaryFolder)
+                .hadoopConfiguration(configuration);
+    }
+
+    private static void ingestRecords(RecordGenerator.RecordListAndSchema recordListAndSchema,
+                                      IngestCoordinatorTestParameters ingestCoordinatorTestParameters,
+                                      TestIngestType ingestType,
+                                      Consumer<ArrowRecordBatchFactory.Builder<RecordList>> arrowConfig) throws Exception {
+        try (IngestCoordinator<RecordList> ingestCoordinator =
+                     createIngestCoordinator(ingestCoordinatorTestParameters, ingestType, arrowConfig)) {
+            for (RecordList recordList : buildScrambledRecordLists(recordListAndSchema)) {
+                ingestCoordinator.write(recordList);
+            }
+        }
+    }
+
+    private static IngestCoordinator<RecordList> createIngestCoordinator(IngestCoordinatorTestParameters parameters,
+                                                                         TestIngestType ingestType,
+                                                                         Consumer<ArrowRecordBatchFactory.Builder<RecordList>> arrowConfig) {
+        return ingestCoordinatorDirectWriteBackedByArrow(parameters, ingestType.getFilePrefix(parameters),
+                arrowConfig, new ArrowRecordWriterAcceptingRecordList());
+    }
+
+    private static void assertThatOrderedRecordsInRange(List<Record> records, LongStream range) {
```

Comment:
Can the method in the other test file be extracted to a helper class so that both test classes can use the same version of this?